lib/setup.py

```diff
@@ -98,19 +92,42 @@ def run(self):
 setuptools.setup(
     name=NAME,
     version=VERSION,
-    description=DESCRIPTION,
-    long_description=LONG_DESCRIPTION,
+    description="The fastest way to build data apps in Python",
+    long_description=(THIS_DIRECTORY / ".." / "README.md").read_text(),
+    long_description_content_type="text/markdown",
     url="https://streamlit.io",
     project_urls={
-        "Source": "https://github.com/streamlit/streamlit",
+        "Source Code": "https://github.com/streamlit/streamlit",
+        "Bug Tracker": "https://github.com/streamlit/streamlit/issues",
+        "Release notes": "https://docs.streamlit.io/library/changelog",
+        "Documentation": "https://docs.streamlit.io/",
+        "Community": "https://discuss.streamlit.io/",
     },
     author="Streamlit Inc",
     author_email="hello@streamlit.io",
+    license="Apache License 2.0",
```

Comment:
I updated this field because the current value is not correctly recognized by Pypi. I set a value that uses Apache Airflow because I couldn't find any description of the standard and the value from Apache Airflow works.

For Streamlit, this field in pypi is as below
<img width="435" alt="Screenshot 2022-09-23 at 01 01 49" src="https://user-images.githubusercontent.com/78743291/191866142-0bbfca0d-c0ca-422b-8787-38936a3d3de7.png">

For Apache Airflow, this field in pypi is as below
<img width="485" alt="Screenshot 2022-09-23 at 01 01 57" src="https://user-images.githubusercontent.com/78743291/191866145-e1d7e350-e562-485e-aeea-0fab31fdbddf.png">